openlibrary/plugins/openlibrary/js/autocomplete.js

```diff
@@ -1,6 +1,6 @@
 // jquery plugins to provide author and language autocompletes.
 
-;(function($) {
+(function($) {
```

Comment:
see comment above

Reply:
okay, I have added `;` back and updated the PR.